builtin/providers/aws/resource_aws_config_configuration_recorder_status.go

```diff
@@ -0,0 +1,115 @@
+		Delete: resourceAwsConfigConfigurationRecorderStatusDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				d.Set("name", d.Id())
+				return []*schema.ResourceData{d}, nil
+			},
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"is_enabled": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceAwsConfigConfigurationRecorderStatusPut(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).configconn
+
+	name := d.Get("name").(string)
+	d.SetId(name)
+
+	if d.HasChange("is_enabled") {
+		isEnabled := d.Get("is_enabled").(bool)
+		if isEnabled {
+			log.Printf("[DEBUG] Starting AWSConfig Configuration recorder %q", name)
+			startInput := configservice.StartConfigurationRecorderInput{
+				ConfigurationRecorderName: aws.String(name),
+			}
+			_, err := conn.StartConfigurationRecorder(&startInput)
+			if err != nil {
+				return fmt.Errorf("Failed to start Configuration Recorder: %s", err)
+			}
+		} else {
+			log.Printf("[DEBUG] Stopping AWSConfig Configuration recorder %q", name)
+			stopInput := configservice.StopConfigurationRecorderInput{
+				ConfigurationRecorderName: aws.String(name),
+			}
+			_, err := conn.StopConfigurationRecorder(&stopInput)
+			if err != nil {
+				return fmt.Errorf("Failed to stop Configuration Recorder: %s", err)
+			}
+		}
+	}
+
+	return resourceAwsConfigConfigurationRecorderStatusRead(d, meta)
+}
+
+func resourceAwsConfigConfigurationRecorderStatusRead(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).configconn
+
+	name := d.Get("name").(string)
+	statusInput := configservice.DescribeConfigurationRecorderStatusInput{
+		ConfigurationRecorderNames: []*string{aws.String(name)},
+	}
+	statusOut, err := conn.DescribeConfigurationRecorderStatus(&statusInput)
+	if err != nil {
+		if awsErr, ok := err.(awserr.Error); ok {
+			if awsErr.Code() == "NoSuchConfigurationRecorderException" {
+				log.Printf("[WARN] Configuration Recorder (status) %q is gone (NoSuchConfigurationRecorderException)", name)
+				d.SetId("")
+				return nil
+			}
+		}
+		return fmt.Errorf("Failed describing Configuration Recorder %q status: %s",
+			name, err)
+	}
+	if len(statusOut.ConfigurationRecordersStatus) < 1 {
+		log.Printf("[WARN] Configuration Recorder (status) %q is gone (no recorders found)", name)
+		d.SetId("")
+		return nil
+	}
+
+	d.Set("is_enabled", statusOut.ConfigurationRecordersStatus[0].Recording)
+
+	return nil
+}
+
+func resourceAwsConfigConfigurationRecorderStatusDelete(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).configconn
+	input := configservice.StopConfigurationRecorderInput{
+		ConfigurationRecorderName: aws.String(d.Get("name").(string)),
+	}
+	_, err := conn.StopConfigurationRecorder(&input)
```

Comment:
The assumption is that `aws_config_configuration_recorder_status` is only responsible for managing the status (stopped/running) of the recorder, whereas `aws_config_configuration_recorder` is responsible for the CRUD of the recorder - as explained in the docs.

So it shouldn't.